src/pages/first-byte-page/FirstBytePage.tsx

```diff
@@ -15,8 +17,14 @@ const FirstBytePage: FC = () => {
         styles={{ width: "600px", height: "400px" }}
       >
         <p>This window has custom width and height.</p>
-        <button>Click Me</button>
       </Window>
+      <div className="fb-sponsors">
+        <Folder
+          sponsorName="mastercard"
+          sponsorLogo="https://media.discordapp.net/attachments/859197594831552542/1296256765914120192/image.png?ex=6711a0b0&is=67104f30&hm=756467d14d2fbabd2494a0f4cfa8769cad9f67e8db7dea38c71c62976bed9991&=&format=webp&quality=lossless&width=712&height=471"
+        ></Folder>
```

Comment:
If I'm not wrong, I think we have the image of master card in our repo. I think we can maybe have the backup as the master card in our repo??? 

But it's not too big of a deal